libs/my-package/project.json

```diff
@@ -25,8 +25,10 @@
       }
     },
     "test": {
-      "executor": "@nx/vite:test",
```

Comment:
`@nx/vite:test` does not seem to support browser mode.

It would be nice to use the Nx vitest plugin instead of the manual command I've added/